composer.json

```diff
@@ -16,20 +22,25 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.7|~3.2|~4.0",
+        "php": ">=7.1",
+        "symfony/framework-bundle": "^4.3.7 || ^5.0",
         "gedmo/doctrine-extensions": "^2.3.4"
     },
     "require-dev": {
         "roave/security-advisories": "dev-master",
-        "symfony/phpunit-bridge": "^4.0",
-        "symfony/security-bundle": "^2.7 || ^3.2 || ^4.0"
+        "symfony/phpunit-bridge": "^4.3.7 || ^5.0",
```

Comment:
```suggestion
        "symfony/phpunit-bridge": "^5.0",
```

You can use the last one, [it is independent from the other packages](https://symfony.com/doc/current/components/phpunit_bridge.html#installation).